mappings/net/minecraft/block/MapColor.mapping

```diff
@@ -71,3 +76,23 @@ CLASS net/minecraft/class_3620 net/minecraft/block/MapColor
 		ARG 1 id
 		ARG 2 color
 	METHOD method_15820 getRenderColor (Lnet/minecraft/class_3620$class_6594;)I
+		ARG 1 brightness
+	METHOD method_38479 validateAndGet (I)Lnet/minecraft/class_3620;
```

Comment:
```suggestion
	METHOD method_38479 get (I)Lnet/minecraft/class_3620;
```
These renames of `get` and `getUnchecked` aims to let users use `get` more often, as `get` handles the ioobe message more gracefully than the direct array access one.